src/models/transactions/transaction.ts

```diff
@@ -46,3 +48,12 @@ export interface TransactionAndMetadata {
   transaction: Transaction
   metadata: Metadata
 }
+
+export {
+  AccountSetFlags,
+  AccountSetTransactionFlags,
+  OfferCreateTransactionFlags,
+  PaymentTransactionFlags,
+  PaymentChannelClaimTransactionFlags,
+  TrustSetTransactionFlags
+}
```

Comment:
Should these be exported here, or in the separate transaction type files?

Reply:
We can export wherever we want to.

I put them in here because it was easiest and it made sense, but I'm open to putting them somewhere else.